pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -473,6 +473,10 @@ bootc switch --mutate-in-place --transport %s %s
 %%end
 `, targetContainerTransport, p.containerSpec.LocalName)
 
+	if p.Kickstart.UserFile != nil {
+		hardcodedKickstartBits += "\n" + p.Kickstart.UserFile.Contents
```

Comment:
Hmm, doesn't that mean that the user cannot override partitioning? Does it mean that we need to use just the user-provided kickstart ig given and skip our parts?

Reply:
Well, _it's complicated and it depends_.

I see a few ways out of this:
1. Document what we add to the kickstart and how the user's is added so they can do their own bits and figure it out themselves.
2. Document what we add to the kickstart and tell users that if they add a kickstart they need to add or replace the bits we add because their file will replace ours.
3. Use the same options we do in the other image types (like edge installer) so the user can control what gets added by enabling or disabling the `Unattended` option and choose if they want to have that or not.